easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.1.2-foss-2020b-TensorFlow-2.5.0.eb

```diff
@@ -0,0 +1,186 @@
+easyblock = 'PythonBundle'
+
+name = 'AlphaFold'
+version = '2.1.2'
+local_tf_version = '2.5.0'
+versionsuffix = '-TensorFlow-{}'.format(local_tf_version)
+
+homepage = 'https://deepmind.com/research/case-studies/alphafold'
+description = "AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known"
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+builddependencies = [
+    # CMake/Doxygen/SWIG are required for building OpenMM
+    ('CMake', '3.18.4', ),
```

Comment:
There's a couple of minor code style issues I'd like to see resolved, like these trailing commas inside the tuple here.

@ThomasHoffmann77 I'll try and look into that today, and send a PR to your branch to propose those changes (rather than commenting on them one by one, which would be double work essentially)...

Reply:
@ThomasHoffmann77 suggested changes in https://github.com/ThomasHoffmann77/easybuild-easyconfigs/pull/1